### PR TITLE
clean up specs and fixtures, remove erroring template

### PR DESCRIPTION
**Pull request by ericcf**

### CODE REVIEW

File: bin/rails [hunk 1]
```diff
@@ -1,4 +1,16 @@
 #!/usr/bin/env ruby
-APP_PATH = File.expand_path('../../config/application',  __FILE__)
-require_relative '../config/boot'
-require 'rails/commands'
+#
```
Comment (michaelwehrley) [inline bin/rails]: why did you change this file and bin/rake?

**Comment by michaelwehrley** [comment]
:+1:

File: bin/rails (hunk 1 above)
Comment (ericcf) [inline bin/rails]: regenerated them because of Rails gem updates
